01_fib/hsy3418/main_test.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote(replaceSpaceWithlineBreak("1 1 2 3 5 8 13 "))
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected, actual)
+	}
+}
+
+func replaceSpaceWithlineBreak(line string) string {
```

Comment:
In Go, it is not so common to have single line functions like this one, particularly if it is called only once like this one is.

It would be just as readable to say above:
```
expected := strconv.Quote(strings.Replace("1 1 2 3 5 8 13 ", " ", "\n", -1))
```
as that is quite clear what it is doing. (also see my other comment about the `-1` argument).

But since I am suggesting elsewhere that you add more tests, it would be ok to keep this function if you prefer that.